src/main/java/org/generationcp/middleware/service/api/study/germplasm/source/GermplasmStudySourceDto.java

```diff
@@ -0,0 +1,151 @@
+package org.generationcp.middleware.service.api.study.germplasm.source;
+
+import org.pojomatic.Pojomatic;
+import org.pojomatic.annotations.AutoProperty;
+
+@AutoProperty
+public class GermplasmStudySourceDto {
+
+	private Integer sourceId;
+	private Integer gid;
+	private Integer groupId;
+	private String designation;
+	private String cross;
+	private Integer lots;
+	private String breedingMethodAbbreviation;
+	private String breedingMethodName;
+	private String breedingMethodType;
+	private String location;
```

Comment:
should this be locationName?

Reply:
This is actually the breeding location. I'll just rename this to breedingLocationName.